index.html

```diff
@@ -2678,6 +2690,42 @@ <h2>Non-DID Identifiers</h2>
 	
 	</section>
 
+	<section id="security-cycles-resolution">
+		<h2>Cycles in Resolution Chains</h2>
+
+		<p>When <a>DID resolver</a> clients dereference identifiers and linked resources in DID documents —
+			especially fields like <code>verificationMethod</code>, <code>controller</code>,
+			or <code>alsoKnownAs</code> — they may encounter <strong>cyclic resolution chains</strong>.
+			These can occur when a <a>DID document</a> references another DID (or URL) that eventually leads
+			back to a previously dereferenced identifier, forming a loop. <a>DID resolvers</a> can
+			also encounter such situations when dereferencing <a>DID URLs</a> that reference
+			<a>service endpoints</a>.</p>
+
+		<div class="example" title="Cycle Through Controllers">
+			<pre><code>did:example:alice
+	└── verificationMethod.controller → did:example:bob
+		└── verificationMethod.controller → did:example:alice</code></pre>
+		</div>
+
+		<p><strong><a>DID resolvers</a> and their clients that perform recursive dereferencing
+			should expect, detect, and handle such cycles</strong>.</p>
```

Comment:
```suggestion
		<p><strong><a>DID resolvers</a> and their clients that perform recursive dereferencing
			expect, detect, and handle such cycles</strong>.</p>
```

Reply:
RECOMMENDED is an RFC term: https://www.ietf.org/rfc/rfc2119.txt

You could use "are expected to" or any variation if you want. Usually just stating it as a fact (like above) is acceptably strong while not invoking normative language that requires testing... though, in this case, we probably do want to test the behavior if there is normative language in the spec around dereferencing in the resolver (which I do think there is)? In any case, don't put normative text in security or privacy considerations.

Reply:
Thanks -- added that to improve readability.